homeassistant/components/media_player/hdmi_cec.py

```diff
@@ -26,9 +26,16 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     """Find and return HDMI devices as +switches."""
     if ATTR_NEW in discovery_info:
         _LOGGER.info("Setting up HDMI devices %s", discovery_info[ATTR_NEW])
-        add_entities(CecPlayerDevice(hass, hass.data.get(device),
-                                     hass.data.get(device).logical_address) for
-                     device in discovery_info[ATTR_NEW])
+        entities = []
+        for device in discovery_info[ATTR_NEW]:
+            hdmi_device = hass.data.get(device)
+            entities.append(CecPlayerDevice(
+                hass, hdmi_device, hdmi_device.logical_address,
+            ))
+        add_entities(entities)
+        for entity in entities:
+            entity.update()
+    return True
```

Comment:
Nothing is checking this return value. Please remove the statement.

Reply:
thnx